modules/microshift-greenboot-microshift-health-script.adoc

```diff
@@ -38,6 +38,6 @@ The `40_microshift_running_check.sh` health check script only performs validatio
 [id="validation-wait-period_{context}"]
 == Validation wait period
 
-The wait period in each validation is five minutes by default. After the wait period, if the validation has not succeeded, it is declared a failure. This wait period is incrementally increased by the base wait period after each boot in the verification loop.
+The wait period in each validation is 10 minutes by default. After the wait period, if the validation has not succeeded, it is declared a failure. This wait period is incrementally increased by the base wait period after each boot in the verification loop.
 
-* You can override the base-time wait period by setting the `MICROSHIFT_WAIT_TIMEOUT_SEC` environment variable in the `/etc/greenboot/greenboot.conf` configuration file. For example, you can change the wait time to three minutes by resetting the value to 180 seconds, such as `MICROSHIFT_WAIT_TIMEOUT_SEC=180`.
+* You can override the base-time wait period by setting the `MICROSHIFT_WAIT_TIMEOUT_SEC` environment variable in the `/etc/greenboot/greenboot.conf` configuration file. For example, you can change the wait time to 5 minutes by resetting the value to 300 seconds, such as `MICROSHIFT_WAIT_TIMEOUT_SEC=300`.
```

Comment:
Should probably spell out the 5, as it is less than 10. I don't see it in our style guide. But it is common practice and what was there before. 
```suggestion
* You can override the base-time wait period by setting the `MICROSHIFT_WAIT_TIMEOUT_SEC` environment variable in the `/etc/greenboot/greenboot.conf` configuration file. For example, you can change the wait time to five minutes by resetting the value to 300 seconds, such as `MICROSHIFT_WAIT_TIMEOUT_SEC=300`.
```

Reply:
Actually, yes, that is AP style, which I was using here, but somewhere recently I saw numerals for all. Hmmm where was that?

Reply:
Ah, here, https://www.ibm.com/docs/en/ibm-style?topic=measurement-numerals-versus-words, so IBM is flexible, but I think OCP was leaning numbers only.